credentialz/gnsi-credentialz.yang

```diff
@@ -95,6 +94,26 @@ module gnsi-ssh {
         }
     }
 
+    // System role console related definitions.
+
+    grouping user-console-credentials-version {
+        description
+            "System role credentials freshness-related data.";
+
+        leaf password-version {
+            type version;
+            description
+                "The version of the password that is currently used to
+                authenticate this user account.";
+        }
```

Comment:
(nit) blank line between these two `leaf` definitions would match the general style in OC models.